pages/studentPortal/labs/multiplication-connect/Index.tsx

```diff
@@ -1,31 +1,21 @@
-import React, { FC, useState } from "react";
+import { indexOf } from "lodash";
+import React, { FC, useState, useEffect } from "react";
 import DiceSection from "../../../../components/math/multiplicationConnect/DiceSection";
 import GameBoard from "../../../../components/math/multiplicationConnect/GameBoard";
 import GameBoardBlock from "../../../../components/math/multiplicationConnect/GameBoardBlock";
 import PlayerSection from "../../../../components/math/multiplicationConnect/PlayerSection";
 import { getRandomItemFromArray } from "../../../api/random";
 
-// todo: convert to useEffect hook for initialization
-const createGrid = () => {
-  let arr = [];
-  let grid = [];
-  for (let i = 4; i < 25; i++) {
-    i % 2 === 0 ? arr.push(i) : "";
-  }
-  for (let i = 0; i < 35; i++) {
-    let gridNumber = getRandomItemFromArray(arr);
-    grid.push({
-      id: i,
-      gridNumber: gridNumber,
-      isSelected: false,
-    });
-  }
-  return grid;
-};
-
-export const calculateWinner = (grid: GameBoardBlock[]) => {
-  // Algorithm ran on block click to determine win
+enum SelectedBy {
+  Unselected = "UNSELECTED",
+  PlayerOne = "PLAYERONE",
+  PlayerTwo = "PLAYERTWO",
+}
 
+export const calculateWinner = (
```

Comment:
Right now this function returns void. To make it more testable it should return a string and you can write a test to assert that the return value is what you expect for a given input test case.

If you're still planning to change the logic for this calculation then it's fine to add the unit test later. But if this is the expected logic then you should add the unit test in your PR. That is the best practice that most engineering teams follow. 

At duolingo, there was a rule that almost every PR had to have at least one accompanying test.

In interviews you should say that you do that to make sure your code is always working as expected.

If you add tests now and you made a mistake in your logic then your PR would fail. If you add tests later, you could be making a mistake now and not catch it until much later.

Not a big deal for this project, but in general the longer it takes for a business to identify a bug the more costly that mistake can be.

It's easiest to fix bugs before the code is merged into main vs after you've merged and started thinking about your next task.

I don't think you have a bug in your code, but this defensive mentality is the mindset you need to have for interviews.